docs/getting_started/projects.md

```diff
@@ -1,4 +1,176 @@
+
+dcrwallet can operate in two distinct modes.
+The first (and default) is to communicate with a single trusted dcrd instance using JSON-RPC.
+The second is a privacy-preserving
+[Simplified Payment Verification (SPV)](https://docs.decred.org/wallets/spv/).
+
+---
+
+## decrediton
+
+- [decred/decrediton](https://github.com/decred/decrediton) (Node.js)
+
+decrediton is a cross-platform GUI wallet for Decred built with Electron.
+It operates by communicating with dcrd and dcrwallet over RPC.
+It also integrates with Politeia to allow ticket holders to vote on Proposals.
+
+---
+
+## Lightning Network
+
+- [decred/dcrlnd](https://github.com/decred/dcrlnd) (Go)
+- [decred/lightning-faucet](https://github.com/decred/lightning-faucet) (Go)
+- [decred/lightning-onion](https://github.com/decred/lightning-onion) (Go)
+
+Decred’s Lightning Network implementation (dcrlnd) is based on the lnd implementation
+for Bitcoin, and is under active development.
+
+---
+
+## dcrdata
+
+- [decred/dcrdata](https://github.com/decred/dcrdata) (Go)
+
+dcrdata is a Decred block explorer, implemented from scratch by the Decred project.
+
+---
+
+## atomicswap
+
+- [decred/atomicswap](https://github.com/decred/atomicswap) (Go)
+
+The atomicswap repository contains utilities to manually perform cross-chain atomic
+swaps between various supported pairs of cryptocurrencies.
+
+---
+
+## Politeia
+
+- [decred/politeia](https://github.com/decred/politeia) (Go)
+- [decred/politeiagui](https://github.com/decred/politeiagui) (Go)
+- [decred/snew-classic-ui](https://github.com/decred/snew-classic-ui) (Go)
+
+Politeia is Decred's solution for decentralized governance.
+[decred/politeia](https://github.com/decred/politeia) implements
+a time-stamped and versioned data store based on git,
+and this is used as the back-end for Politeia.
+The front-end is implemented in [decred/politeiagui](https://github.com/decred/politeiagui)
+and [decred/snew-classic-ui](https://github.com/decred/snew-classic-ui).
+
+---
+
+## dcrtime
+
+- [decred/dcrtime](https://github.com/decred/dcrtime) (Go)
+- [decred/dcrtimegui](https://github.com/decred/dcrtimegui) (Node.js)
+
+dcrtime is a blockchain based timestamping service,
+inspired by the work of Peter Todd’s
+[OpenTimestamps](https://petertodd.org/2016/opentimestamps-announcement).
+dcrtime allows a nearly unlimited number of hashes
+to be timestamped onchain with the inclusion of a
+single merkle root in a transaction.
+
+dcrtimegui implements a front-end for dcrtime,
+enabling files to be timestamped through a web-browser.
+
+---
+
+## Mobile Wallets
+
+- [raedahgroup/dcrios](https://github.com/raedahgroup/dcrios) (Swift)
+- [decred/dcrandroid](https://github.com/decred/dcrandroid) (Java, Kotlin)
+
+Native wallets for both iOS and Android.
+
+---
+
+## dcrstakepool
+
+- [decred/dcrstakepool](https://github.com/decred/dcrstakepool) (Go)
+
+dcrstakepool is the reference implementation for a Decred VSP
```

Comment:
VSP (Voting Service Provider) -> Voting Service Provider (VSP)

We're not totally consistent on this, but I think we do the latter more often